browser/create-karma-conf.js

```diff
@@ -13,7 +13,13 @@ module.exports = function (files) {
 			browserify: {
 				debug: true
 			},
-			browsers: [process.env.CI ? 'PhantomJS' : 'Chrome'],
+			customLaunchers: {
+				ChromeHeadlessNoSandbox: {
+					base: 'ChromeHeadless',
+					flags: ['--no-sandbox']
+				}
+			},
+			browsers: [process.env.CI ? 'ChromeHeadlessNoSandbox' : 'ChromeHeadless'],
```

Comment:
The `--no-sandbox` is necessary for Travis CI https://docs.travis-ci.com/user/chrome.